endpoints-control/src/main/java/com/google/api/control/Client.java

```diff
@@ -266,17 +261,16 @@ public void report(ReportRequest req) {
         transport.services().report(serviceName, req).execute();
         statistics.totalTransportedReportTimeMillis.addAndGet(w.elapsed(TimeUnit.MILLISECONDS));
       } catch (IOException e) {
-        log.log(Level.SEVERE,
-            String.format("direct send of a report request %s failed because of %s", req, e));
+        log.atSevere().withCause(e)
+            .log("direct send of a report request %s failed because of %s", req);
       }
     }
 
     if (isRunningSchedulerDirectly()) {
       try {
         scheduler.run(false /* don't block */);
       } catch (InterruptedException e) {
-        log.log(Level.SEVERE,
-            String.format("direct run of scheduler failed because of %s", e));
+        log.atSevere().withCause(e).log("direct run of scheduler failed because of %s");
```

Comment:
There's no value for the `%s` placeholder in this one; the exception was being used as the value in the old version.